### ISSUE TITLE
Ignore positions don't work in recursion

### ISSUE THREAD

**Issue by schnaader** [schnaader/precomp-cpp#8]
At the moment, you can set a list of ignore positions using the ' -i' parameter. However, when in recursion, positions are reset to 0 so a given position like `1234` can occur several times at different recursion levels which is unwanted behaviour - the user wants to ignore a certain stream and nothing more.

Proposal: To prevent this, the stream position output in verbose mode should output positions in the format `xxxx_yyyy_zzzz...` - every recursion adds an underscore to the position, so e.g. a stream starting at position 1234 that contains a stream starting at position 567 in it would lead to the position `1234_567`. The ignore list parameter parsing and usage in the detection would have to be changed accordingly.

**Comment by BilawalAhmed0900** [comment]
Should the ignore list be read as comma seperate values?

Example -i11,22,33 will ignore the position 33 at depth 2 reached from position 22 at depth 1 reached from position 11 at depth 0.

**Comment by schnaader** [comment]
Yes, additionally enabling comma separation would be useful. With the current syntax, multiple -i parameters have to be used, which is cumbersome (e.g. -i11 -i22 -i33).
